Docs/sphinx_doc/VerticalSCoord.rst

```diff
@@ -0,0 +1,93 @@
+
+* The **true** sigma-coordinate system is recovered as :math:'h_c \rightarrow \infty '. This is useful when configuring applications with **flat** bathymetry and **uniform** level thickness. Practically, you can achieve this by setting Tcline to **1.0d+16** in roms.in. This will set :math:'h_c =1.0\times 10^{16}'. Although not necessary, we also recommend that you set :math:'\theta _S = 0' and :math:'\theta _B =0'.
+
+In an undisturbed ocean state, :math:'\zeta \equiv 0', this transformation yields the following unperturbed depths, :math:'\hat{z}',
+
+.. math::
+   \hat{z} \left( x,y,\sigma \right) \equiv h\left( x,y\right) S\left( x,y,\sigma \right) =h\left( x,y\right) \left[ \frac{h_c \sigma +h\left( x,y\right) C\left( \sigma \right)}{h_c +h\left( x,y\right) } \right]
+
+and
+
+.. math::
+   d\hat{z} =d\sigma h\left( x,y\right) \left[ \frac{h_c}{h_c +h\left( x,y\right) } \right]
+
+As a consequence, the uppermost grid box retains very little dependency from bathymetry in deep areas, where :math:'h_c \ll h\left( x,y\right) '. For example if :math:'h_c =250m' and :math:'h\left( x,y\right) ' changes from :math:'2000' to :math:'6000m', the uppermost grid box changes only by a factor of 1.08 (less than 10%).
+
+
+
+Vertical Stretching
+-------------------
+
+The above generic vertical transformation design facilitates numerous vertical stretching functions :math:'C\left( \sigma \right) '. This function is defined in terms of several parameters which are specified in the standard input file, roms.in.
+
+Stretching Function Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :math:'C\left( \sigma \right) ' is a dimensionless, nonlinear, monotonic function;
+* :math:'C\left( \sigma \right) ' is a continuous differentiable function, or a differentiable piecewise function with smooth transition;
+* :math:'C\left( \sigma \right) ' must be discretized in terms of the fractional stretched vertical coordinate :math:'\sigma ',
+
+  .. math::
+     \sigma \left( k \right) = \begin{cases}{\frac{k-N}{N}, & \text{at vertical }W\text{-points}, & k=0,\ldots ,N,}\\{\frac{k-N-0.5}{N}, & \text{at vertical}\rho \text{-points}, & k=1,\ldots ,N} \end{cases}
+
+* :math:'C\left( \sigma \right) ' must be constrained by :math:'-1 \leq C\left( \sigma \right) \leq 0', that is,
+
+  .. math::
+     C\left( \sigma \right) = \begin{cases}{0, & \text{if } \sigma = 0, & C\left( \sigma \right) = 0, & \text{at the free-surface};}\\{-1, & \text{if } \sigma = -1, & C\left( \sigma \right) = -1, & \text{at the ocean bottom}.} \end{cases}
+
+Available Stretching Functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+4. A. Shchepetkin (2010) UCLA-ROMS current function, Vstretching=4. :math:'C\left( \sigma \right) ' is defined as a continuous, double stretching function:
```

Comment:
consider making this and the transformation numbering (and any references to ROMS numbering) consistent in this doc